cpp/zotaut/dump_delivered_marc_records.cc

```diff
@@ -79,19 +80,22 @@ void GetJournalEntriesFromDb(DbConnection * const db_connection, const std::stri
             const auto dois(record.getDOIs());
             csv_row += ";" + TextUtil::CSVEscape(StringUtil::Join(dois, '\n'));
 
-            for (const auto &_936_field : record.getTagRange("936")) {
-                csv_row += ";" + _936_field.getFirstSubfieldWithCode('j');
-                csv_row += ";" + _936_field.getFirstSubfieldWithCode('d');
-                csv_row += ";" + _936_field.getFirstSubfieldWithCode('e');
-                break;
-            }
+            const auto issue_info(BSZUtil::ExtractYearVolumeIssue(record));
+
+            if (not issue_info.year_.empty())
+                csv_row += ";" + issue_info.year_;
```

Comment:
This could be a problem. Since we're exporting a CSV file here, the field delimiter `;` should always be appended even if the value is empty, else the amount of columns per csv_row will vary